diff --git a/node_modules/ladda/js/ladda.js b/node_modules/ladda/js/ladda.js
index 6362455..8072fd0 100644
--- a/node_modules/ladda/js/ladda.js
+++ b/node_modules/ladda/js/ladda.js
@@ -67,7 +67,6 @@ export function create(button) {
                 spinner = createSpinner(button);
             }
 
-            button.disabled = true;
             button.setAttribute('data-loading', '');
 
             clearTimeout(timer);
@@ -93,7 +92,6 @@ export function create(button) {
          */
         stop: function() {
             if (instance.isLoading()) {
-                button.disabled = false;
                 button.removeAttribute('data-loading');
             }
 
@@ -147,7 +145,6 @@ export function create(button) {
 
         remove: function() {
             clearTimeout(timer);
-            button.disabled = false;
             button.removeAttribute('data-loading');
 
             if (spinner) {
